docs/source/eventing/actions/label.rst

```diff
@@ -24,28 +24,33 @@ Changes the text and color of a generic label device.
 +----------+------------------+---------------------+---------------------------------------------------+
 | COLOR    | |no|             | |no|                | New color                                         |
 +----------+------------------+---------------------+---------------------------------------------------+
-| BLINK    | |no|             | |no|                | Blinking on or off                                |
-+----------+------------------+---------------------+---------------------------------------------------+
 | FOR      | |no|             | |no|                | | Determine how long this new label lasts         |
 |          |                  |                     | | before we change back to the previous label     |
 +----------+------------------+---------------------+---------------------------------------------------+
 | AFTER    | |no|             | |no|                | After how long do we want the new label to be set |
 +----------+------------------+---------------------+---------------------------------------------------+
 
-.. note:: 1. Valid values for ``BLINK``
-
-   - on
-   - off
-
-
-2. Units for ``FOR`` and ``AFTER``
+.. note:: Units for ``FOR`` and ``AFTER``
 
    - MILLISECOND
    - SECOND
    - MINUTE
    - HOUR
    - DAY
 
+.. versionadded:: 8.0
+
++----------+------------------+---------------------+---------------------------------------------------+
+| **Name** | **Required**     | **Multiple Values** | **Description**                                   |
++----------+------------------+---------------------+---------------------------------------------------+
+| BLINK    | |no|             | |no|                | Blinking label text on or off                     |
```

Comment:
I would remove that note below and replace the description with:
"Blinking label text `on` or `off`"

Reply:
Done